kolibri/plugins/coach/assets/src/views/plan/LessonContentPreviewPage/index.vue

```diff
@@ -173,6 +190,11 @@
         );
       },
     },
+    watch: {
+      isSelected() {
```

Comment:
This watcher is not currently used though I added it because I would expect that `disableSelectButton` will be appropriately reset on selection state change.